trip-to-carbon.ts

```diff
@@ -23,10 +24,12 @@ type CarbonFootprintOptions = CarbonFootprintBaseOptions & (
   CarbonFootprintFuelOptions
 )
 
-export async function carbonFootprint(options: CarbonFootprintOptions): Promise<number> {
+export async function carbonFootprint (options: CarbonFootprintOptions): Promise<number> {
   const baseUrl = 'baseUrl' in options ? options.baseUrl : 'https://api.triptocarbon.xyz'
-
   const params = new URLSearchParams()
+
+  if ('id' in options) params.set('id', options.id!)
```

Comment:
Why does `options.id!` need a `!` at the end? Could this be `params.set('id', options.id)`?

Reply:
Ah alright, I've never actually used typescript. I'll change it.